jekyll/_includes/site/analysisfield.html

```diff
@@ -15,5 +15,5 @@
 </script>
 
 <script type="text/x-template" data-template="readingLevel">
-  <p class="{level}"><a href="https://en.wikipedia.org/wiki/Flesch%E2%80%93Kincaid_readability_tests" target="_blank">Grade reading level</a>:<span>{readingLevel}</span></p>
+  <p class="{level}">Grade reading level:<span>{readingLevel}</span></p>
```

Comment:
I feel like we should link to something to explain why this is important, if we're not explaining it on the site.